tests/sphinxext/sources/workchain/conf.py

```diff
@@ -86,7 +86,7 @@
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-html_theme = 'sphinx_rtd_theme'
+html_theme = 'alabaster'
```

Comment:
why change this?

Reply:
Guess this is now the only place where we need `sphinx_rtd_theme`..

Reply:
because `sphinx_rtd_theme` is a separate theme package that we no longer have as a dependency, whereas alabaster if the built-in theme

Reply:
Thanks for the clarification!